docs/content/preview/drivers-orms/python/yugabyte-psycopg2.md

```diff
@@ -128,8 +128,8 @@ The following table describes the connection parameters required to connect, inc
 | database/dbname | Database name | yugabyte |
 | user | User connecting to the database | yugabyte |
 | password | User password | yugabyte |
-| `load_balance` | [Uniform load balancing](../../smart-drivers/#cluster-aware-load-balancing) | Defaults to upstream driver behavior unless set to 'true' |
```

Comment:
Please check this table and wording with that in [jdbc page](https://docs.yugabyte.com/preview/drivers-orms/java/yugabyte-jdbc/#step-2-set-up-the-database-connection).